store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/ParquetStore.java

```diff
@@ -183,7 +185,7 @@ protected GetAdjacentIdsHandler getAdjacentIdsHandler() {
     @Override
     protected void addAdditionalOperationHandlers() {
         addOperationHandler(GetDataFrameOfElements.class, new GetDataframeOfElementsHandler());
-        addOperationHandler(ImportRDDOfElements.class, new ImportRDDOfElementsHandler());
```

Comment:
The issue doesn't say to remove the handler for `ImportRDDOfElements` and replace it by `ImportJavaRDDOfElements`. Can both be handled?